src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -892,6 +929,51 @@ inline void rpp_store48_f32pkd3_to_f16pkd3_avx(Rpp16f *dstPtr, __m256 *p)
     _mm_storeu_si128((__m128i *)(dstPtr + 45), px128[3]);
 }
 
+inline void rpp_load96_u8pkd3_to_f32pln3_avx512(Rpp8u *srcPtr, __m512 *p)
+{
+    __m512i px[2];
+
+    px[0] = _mm512_loadu_si512((__m512i *)srcPtr);
+    px[1] = _mm512_loadu_si512((__m512i *)(srcPtr + 48));
+    __m512i pxCvt[6];
+
+    __mmask64 maskR = 0x9249249249240000;
```

Comment:
Please add comment stating what this value represent, same for next 2 masks also
If it is constant, please move define this masks outside as const